region/wreckedship/main/Wrecked Ship East Super Room.json

```diff
@@ -166,6 +174,31 @@
       "gModeRegainMobility": {},
       "devNote": "The Covern will do more damage than the Bull, so there is no canRiskPermanentLossOfAccess."
     },
+    {
+      "link": [1, 1],
+      "name": "Come in in R-Mode",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [],
+      "clearsObstacles": ["A"]
+    },
+    {
+      "link": [1, 1],
+      "name": "Coverns and Atomics Farm (Power Off)",
+      "requires": [
+        {"not": "f_DefeatedPhantoon"},
+        "canRiskPermanentLossOfAccess",
+        {"or": [
+          "Ice",
```

Comment:
Ice isn't really strong enough to make the covern farm good, and the atomic farm is still tricky.
Screw only helps with the Coverns.

As an R-Mode farm spot, only the coverns respawn so you would want items which kill them faster.